maquetado/books-list.html

```diff
@@ -0,0 +1,57 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Books list</title>
+  <meta name="description" content="books-list">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <div class="big-container">
+    <div class="books-list-container">
+      <div class="book-preview">
+        <img src ="./assets/book-cover.png" class="book-small-preview">
+        <h1 class="book-preview-info bold">Titulo del libro</h1>
+        <h1 class="book-preview-info detail-color">Autor del libro</h1>
+      </div>
+      <div class="book-preview">
+        <img src ="./assets/book-cover.png" class="book-small-preview">
+        <h1 class="book-preview-info bold">Titulo del libro</h1>
+        <h1 class="book-preview-info detail-color">Autor del libro</h1>
```

Comment:
Same. There should be only one `h1` tag per section (In this case). Ideally you should have only one `h1` tag to describe your current page and `h2` for posts and articles sections.